<!-- -*- mode: XML -*- -->
<!-- run lidar driver and tf and point process -->

<launch>

  <!-- start velodyne_pointcloud nodelet -->
  <include file="$(find velodyne_pointcloud)/launch/VLP16_points.launch">
  </include>


<!--<launch>-->

  <!-- start sick nodelet -->
  <!--<include file="$(find lms1xx)/launch/LMS1xx.launch">
  </include>-->

<!--<launch>-->

  <!-- start velodyne_pointcloud nodelet -->
  <include file="$(find lidar_detect)/launch/lidar_detect_node_launch.launch">
  </include>

  <!-- transform broadcaster-->
  <!--<node name="bslk" pkg="build_baselink" type="bslk" output="screen"/>-->
  <!-- point processor-->
  <!--<node name="point_process" pkg="build_baselink" type="point_process" output="screen"/>-->
  <!-- dynamic reconfigure-->
  <!--<node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" output="screen"/>-->
  

</launch>